# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/slim/paths.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- slim-1.3.6/slim.conf.vanilla	2020-09-30 21:20:38.555570904 +0200
+++ slim-1.3.6/slim.conf	2020-09-30 21:21:17.827077194 +0200
@@ -1,7 +1,7 @@
 # Path, X server and arguments (if needed)
 # Note: -xauth $authfile is automatically appended
 default_path        /bin:/usr/bin:/usr/local/bin
-default_xserver     /usr/bin/X
+default_xserver     /usr/X11/bin/X
 #xserver_arguments   -dpi 75
 
 # Commands for halt, login, etc.
@@ -11,7 +11,7 @@
 #suspend_cmd        /usr/sbin/suspend
 
 # Full path to the xauth binary
-xauth_path         /usr/bin/xauth 
+xauth_path         /usr/X11/bin/xauth
 
 # Xauth file for server
 authfile           /var/run/slim.auth
